### ISSUE TITLE
Lab 10 peer review

### ISSUE THREAD

**Issue by SalvoCab** [s288976/computational_intelligence_23_24#2]
You've written code that's really easy to read and understand.
It's so good to see that you've also implemented human interaction to play matches against the smart player bot.
An interesting implementation could be to add a variable (the value is between 0 and 1) that can decide to perform the "best move" or a random move in order to create slightly less smart players. The value of this variable may decrease as we get closer and closer to the end of the game. It's just an idea for future implementation.
In conclusion, I think you made a smarter player who is really smart, I'm curious to see the winning percentage when the smart player always plays first and also another result compared to when he always plays second.
